package/yast2-rear.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jan 19 11:23:26 UTC 2021 - aabdallah@suse.com
+
+- Don't show a warning anymore for vfat partitions, as versions
+  1.18 and higher of rear support them (bsc#1180599)
+
```

Comment:
Please, mention the new version (4.1.1) in a new line (like the previous of entries do).